src/checks/json.check.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 const status = require('./status');
 const Check = require('./check');
+const fetch = require('node-fetch');
```

Comment:
Then we'd be storing metrics in graphite about how long graphite takes to respond!  (And pingdom).

This doesn't feel like Metrics we need, IMO

I think for everything else it might be wise to have n-fetch or something being an instrumented version of fetch?